pkgs/top-level/stage.nix

```diff
@@ -185,10 +185,25 @@ let
     # that target system. For instance, pkgsCross.raspberryPi.hello,
     # will refer to the "hello" package built for the ARM6-based
     # Raspberry Pi.
+    #
+    # This is essentially the same thing as pkgsLocal except for crossSystem
+    # instead of localSystem.
     pkgsCross = lib.mapAttrs (n: crossSystem:
                               nixpkgsFun { inherit crossSystem; })
                               lib.systems.examples;
 
+    # This maps each entry in lib.systems.examples to its own package set where
+    # packages are built and run on that system.  For example,
+    # pkgsLocal.x86_64-linux.hello will refer to the "hello" package bult for
+    # x86_64-linux.
+    #
+    # This is essentially the same thing as pkgsCross except for localSystem
+    # instead of crossSystem.
+    pkgsLocal =
```

Comment:
You'll need to add this new package set to this exclusion list, or ofborg will infinitely recurse: https://github.com/NixOS/nixpkgs/blob/1b439a273dadd4c675e81a0e05abbc9258889c02/pkgs/top-level/release-attrpaths-superset.nix#L50

(We just ran into this issue with https://github.com/NixOS/nixpkgs/pull/274089 :laughing: )